packages/web/src/pages/sign-up-page/SignUpRootPage.module.css

```diff
@@ -0,0 +1,7 @@
+.container {
+  height: 100%;
+}
+
+.whitePageContainer {
+  height: 100%;
+}
```

Comment:
We can use `<Box w='100%'> or h='100%'` for these and similar uses below

Reply:
ooh thanks!